recommendation/templates/recommendation/recommendation_form.html

```diff
@@ -0,0 +1,44 @@
+{% extends 'dashboard/index.html' %}
```

Comment:
The functionality might be more robust if you extend "recommendation/index.html" instead of extending "dashboard/index.html" since we're already extending from that in index (recommendation). That way, I think you can get rid of the static import statements as well and have them only in your "recommendation/index.html" (help reduce the no. of cdn calls).

Reply:
done